crates/miden-testing/src/tx_context/context.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(unused_imports)]
```

Comment:
@tomyrd Can't quite get around this ATM. Something to do with the async feature flag again no doubt.

See failure: https://github.com/0xMiden/miden-base/actions/runs/15500458627/job/43646935828

```
cargo clippy --no-default-features --target wasm32-unknown-unknown --workspace --lib --exclude bench-prover -- -D warnings
...
error: unused import: `alloc::boxed::Box`
```

Reply:
Can you try changing the import to `#[cfg(feature = "async")]`, i.e. remove `target_family = "wasm"`? That worked for me locally, but 100% sure if all CI checks pass.